docs/dev.rst

```diff
@@ -1,20 +1,522 @@
+
+To run the ``qa_org`` flow against the ``qa`` org:
+
+.. code-block:: console
+
+    $ cci flow run qa_org --org qa
+
+
+Create QA Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Out-of-the-box, the ``config_dev`` and ``config_qa`` flows are the same. Many teams have a requirement for additional configurations to be deployed when performing QA, but not when developing a new feature.
+
+    Example: Salesforce.org teams often modify the ``config_qa`` flow to deploy configurations that pertain to large optional features in a package. These configurations are subsequently tested by the product's robot test suites.
+
+To capture your own QA configurations, spin up a new org...
+
+.. code-block::
+
+    $ cci flow run qa_org
+
+Make the necessary changes, and run:
+
+.. code-block::
+
+    $ cci task run retrieve_qa_config
+
+This task defaults to capturing this metadata under ``unpackaged/config/qa``.
+
+.. note:: The configuration metadata can also be stored in a different location by using the ``--path`` option.
+
+..
+
+To delete the org...
+
+.. code-block:: console
+
+    $ cci org remove qa
+
+Then re-create it...
+
+.. code-block:: console
+
+    $ cci flow run qa_org --org qa
+
+Then run the ``deploy_qa_config`` to deploy the previously captured configurations to the org.
+
+.. code-block:: console
+
+    $ cci task run deploy_qa_config --org qa
+
+To require that the ``qa_org`` flow always runs this task, add a ``deploy_qa_config`` task step under the ``flows__qa_config`` section of the ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    qa_config:
+        steps:
+            3:
+                task: deploy_qa_config
+
+So now ``qa_config`` (which is included in the ``qa_org`` flow) executes the ``deploy_qa_config`` task as the third step in the flow.
+
+
+
+Manage Dependencies
 -------------------
 
-Run tasks and flows from another project
-----------------------------------------
+CumulusCI is built to automate the complexities of dependency management for extension package projects. CumulusCI currently handles these main types of dependencies for projects:
+
+* **Managed Packages**: Require a certain version of a managed package
+* **Unmanaged Metadata**: Require the deployment of unmanaged metadata
+* **GitHub Repository**: Dynamically include the dependencies of another CumulusCI configured project
+
+The ``update_dependencies`` task handles deploying dependencies to a target org, and is included in all flows designed to deploy or install to an org. 
+
+To run the ``update_dependencies`` task: 
+
+.. code-block:: console
+
+    $ cci task run update_dependencies
+
+
+Managed Package Dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Managed package dependencies are rather simple. Under the ``project__dependencies`` section of the ``cumulusci.yml`` file, specify the namespace of the target package, and the required version number.
```

Comment:
Original version was ``project``-->``dependencies``.  Just confirming that ``project__dependencies`` is correct.

Reply:
Correct